File: Classproblem2.java
public class Classproblem2 {
    
    public static void main(String[] args)
    {


        /*Yashika's code, time complexity is O(N3) */
         int n=8;
        for(int i = 0; i <n; i++)
        {
            
            if(i == 3 || i == 4)
                for(int j = 0; j < n; j++)
                    System.out.print("# ");
            else
                for(int j = 0; j < n;j++)
                    if(j % 3 == 2)
                        System.out.print("#");
                    else
                        System.out.print("  ");
            System.out.println("");
}
}
}

File: Classproblem2.cpp
// #include<bits/stdc++.h>
#include<iostream>
using namespace std;
int main(){


    /* Time Complexity is O(N2)*/
    int n=8;
    cout<<"Give me a number: ";
    cin>>n;

    for(int i=1;i<=n;i++){
        for(int j=1;j<=n;j++){
            if (i==4 || i==5 || j%3==0){
                cout<<" * ";
            }
            else{
                cout<<"   ";
            }
        }
        cout<<endl;
    }

}